paywall/src/data-iframe/cacheHandler.js

```diff
@@ -32,7 +32,7 @@ async function _put(window, key, value) {
 }
 
 export async function getKeys(window) {
-  return _get(window, 'keys')
+  return (await _get(window, 'keys')) || {}
```

Comment:
Could we set the default value inside of `_get` rather than here? This decreases the risk of bugs if/when we change the return value...

Reply:
no, because `getAccountBalance` uses `_get` as well, and that should always return a string